data-overview/english.md

```diff
@@ -24,29 +11,18 @@ This feature gives you an overview of the current status of your car / errors an
-| Battery health | So the user can get a warning if the car's battery voltage is low | Saved until the user requests deletion |
-| Warning if the car is towed | So the user can get a warning if the car is towed | 7 days historical data and most recent value |
-| Windshield wiper fluid light | So the user can get a warning if the windshield wiper fluid light is on the car | 7 days historical data and most recent value |
-| Coolant light | So the user can get a warning if the coolant light is on the car | 7 days historical data and most recent value |
-| Brake system light | So the user can get a warning if the brake system light is on the car | 7 days historical data and most recent value |
+| Car service and oil change data | Used for calculation so that the user can see when the car needs service or oil change again | 60 days historical data and most recent value |
+| Battery voltage | So the user can see the car's current battery voltage | Saved until the user requests deletion |
+| Adblue remaining in km* | So the user can see the car's remaining km until next refuel of adblue is needed | 7 days historical data and most recent value |
+| Car dashboard warning lights* | So the user can get a warning about the warning light and advise on how to proceed | 7 days historical data and most recent value |
 | Error codes from car | So the user can get a warning about technical errors on the car that does not trigger warning lights | 7 days historical data and most recent value |
-| Power failure on OBD device | So the user can get a warning if the OBD device in the car is demounted | 7 days historical data and most recent value |
-| Suspected collision | So the user can get a warning if a collision is suspected | 7 days historical data and most recent value |
-| Tire pressure alarm | So the user can get a warning about low tire pressure | 7 days historical data and most recent value |
-| Brake fluid light | So the user can get a warning if the brake fluid light is on the car | 7 days historical data and most recent value |
-| Low oil warning | So the user can get a low oil level warning | 7 days historical data and most recent value |
-| ESP deactivation light | So the user can get a warning if the ESP is disabled or broken | 7 days historical data and most recent value |
-| ESP activation light | So the user can get a warning if the ESP is disabled or broken | 7 days historical data and most recent value |
-| Warning for low battery voltage on the car battery | So the user can get a warning if the car's battery voltage is low | 7 days historical data and most recent value |
-| Warning on changes to the car's battery voltage | So the user can get a warning if there is a change in the car's battery voltage | 7 days historical data and most recent value |
+| Warning for car battery voltage drop | So the user can get a warning if the car's battery voltage suddenly drops | 7 days historical data and most recent value |
```

Comment:
Why only 7 days here? I could imagine support querying into this with a response time over a week. I think it would be better with the standard 60 days below. It will be easier to debug and provide support with explanations having 60 days history. The same goes for all the 7 days intervals.